faq.html

```diff
@@ -28,10 +60,12 @@ <h1>FAQ</h1>
     workflows. Below we include some suggested steps for getting started and answers to some
     frequently asked questions about using GitHub for W3C spec work.
   </p>
-  <section>
-    <h2>Getting Started</h2>
-    <section>
-      <h3>Informal drafts in your personal account</h3>
+
+  <section id="starting">
+    <h2>Getting started <a href="#starting">&#x1F517;</a></h2>
```

Comment:
Any reason for not using the UTF-8 character directly?

Reply:
I use entities sometimes to be sure that every OS, editor, IDE, etc out there will handle the file correctly. Sometimes arcane glyphs are rendered in weird ways, especially in text-based editors. Or the glyph shown takes some extra height or width, or it's rendered as a little square, as *mojibake*, or invisible&hellip;